doc/install.md

```diff
@@ -3,200 +3,286 @@
 ## Table of Contents
 
 - [Create a Kubernetes Cluster](#create-a-kubernetes-cluster)
-- [Install release v0.18.9 and use Coscheduling](#install-release-v0189-and-use-coscheduling)
+- [Install release v0.19.8 and use Coscheduling](#install-release-v0198-and-use-coscheduling)
   - [Test Coscheduling](#test-coscheduling)
 - [Uninstall Scheduler-plugins](#uninstall-scheduler-plugins)
 
 ## Create a Kubernetes Cluster
 
 Firstly you need to have a Kubernetes cluster, and a `kubectl` command-line tool must be configured to communicate with the cluster.
 
-The Kubernetes version must equal to or greater than **v1.18.0**. To check the version, use `kubectl version --short`.
+The Kubernetes version must equal to or greater than **v1.19.0**. To check the version, use `kubectl version --short`.
 
 If you do not have a cluster yet, create one by using one of the following provision tools:
 
 * [kind](https://kind.sigs.k8s.io/docs/)
 * [kubeadm](https://kubernetes.io/docs/admin/kubeadm/)
 * [minikube](https://minikube.sigs.k8s.io/)
 
-## Install release v0.18.9 and use Coscheduling
+## Install release v0.19.8 and use Coscheduling
 
-In this section, we will walk you through how to replace the default scheduler with the scheduler-plugins image. As the new image is built on top of the default scheduler, you won't lose any vanilla Kubernetes scheduling capability. Instead, a lot of extra out-of-box functionalities (implemented by the plugins in this repo) can be obtained, such as coscheduling.
+In this section, we will walk you through how to replace the default scheduler with the
+scheduler-plugins image. As the new image is built on top of the default scheduler, you won't lose
+any vanilla Kubernetes scheduling capability. Instead, a lot of extra out-of-box functionalities
+(implemented by the plugins in this repo) can be obtained, such as coscheduling.
 
-1. Log on Master node 
-   * If your cluster is created by `kind`
-     ```bash
-     sudo docker exec -it $(sudo docker ps | grep control-plane | awk '{print $1}') bash
-     ```
+> The following steps are based on a Kubernetes cluster created by Kind.
+
+1. Log into the control plane node 
+   
+    ```bash
+    sudo docker exec -it $(sudo docker ps | grep control-plane | awk '{print $1}') bash
+    ```
   
-2. Backup `kube-scheduler.yaml`
+1. Backup `kube-scheduler.yaml`
 
```

Comment:
seem the sequence is wrong. It is always `1.`

Reply:
It's intentional, it will be rendered properly.